src/Xamarin.Android.Build.Tasks/Tasks/Legacy/ValidateJavaVersion.cs

```diff
@@ -76,8 +76,6 @@ Version GetJavaVersionForBuildTools ()
 			if (!Version.TryParse (buildToolsVersionString, out buildTools)) {
 				return Version.Parse (LatestSupportedJavaVersion);
 			}
-			if (buildTools >= new Version (30, 0, 0))
```

Comment:
It occurs to me that instead of removing this check entirely, we should instead be "more accurate".  We *know* Build-tools 30.0.0 and 30.0.1 require JDK 11, it's 30.0.2 (and later?) that work with JDK 8.  Thus, should we check for what we know?

```csharp
if (buildTools >= new Version (30, 0, 0) && buildTools < new Version (30, 0 2)) {
    return new Version (11, 0);
}
```

Reply:
Since we redistribute `apksigner.jar` now, both 30.0.0 and 30.0.1 will work.